packages/core/scripts/test/launchTestUtilities.ts

```diff
@@ -189,6 +189,7 @@ async function invokeVSCodeCli(vsCodeExecutablePath: string, args: string[]): Pr
 
     if (spawnResult.status !== 0) {
         console.log('output: %s', spawnResult.output)
+        console.log('error: %s ', spawnResult.error)
```

Comment:
FYI: `%O` gives the full object details in a nice, readable format

```suggestion
        console.log('error: %O', spawnResult.error)
```